cmd/help.go

```diff
@@ -82,8 +82,8 @@ var helpTexts = map[string]helpEntry{
 				Usage: "Do not display the super pretty logo on init.",
 			},
 			cli.StringFlag{
-				Name:  "ipfs-path,P",
-				Usage: "Specify an explicit path to an IPFS repository. Useful if you have more than one.",
+				Name:  "ipfs-path-or-url",
```

Comment:
It would be nice to show an example of expected form of url (and even path).

Reply:
Done. Also renamed to ``--ipfs-path-or-multiaddr`` since it wasn't a URL, but a multiaddr.